docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,611 @@
+```
+
+## Step 4 - Build a Benchmark Configuration File
+The benchmark configuration file defines the benchmark rounds and references the defined workload module(s). It will specify the number of test workers to use when generating the load, the number of test rounds, the duration of each round, the rate control applied to the transaction load during each round, and options relating to monitors. This particular tutorial will not make use of any of the available monitors or transaction observers; for these details please refer to the documentation.
+
+The benchmark configuration file may be provided in a yaml or json format: here we will use a yaml format. Please note that yaml files are case sensitive and all labels are in lowercase. 
+
+The benchmark configuration file has a single required stanza:
+
+``` bash
+test:
+```
+
+### Create A Template Benchmark Configuration File
+
+Under the **benchmarks** folder create a file called **myAssetBenchmark.yaml** with the following content:
+
+```yaml
+test:
+    name: asset-transfer-basic-benchmark
+    description: A test benchmark
+    workers:
+    rounds:
+```
+
+__test__: The root level block that contains the benchmark test information.
+
+__name__: The name of the test, in this case "asset-transfer-basic-benchmark".
+
+__descriptions__: A description for the benchmark, in this case "A test benchmark".
+
+__workers__: A set of keys used to define the number of workers (separate worker client instances) used in the subsequent benchmark.
+
+__rounds__: An array of distinct test rounds that will be progressed sequentially. Rounds may be used to benchmark different smart contract methods, or the same method in a different manner. 
+
+### Populating the Template File
+We will now populate the template file to specify the number of workers and the test round that uses the workload module we have created.
+
+#### Populate Workers
+We will be using two separate workers, this is accomplished through the workers specification:
+
+``` yaml
+  type: local
+  number: 2
+```
+
+#### Populate Rounds
+Each `round` block contains the following: 
+- `label` - the unique header label to use for the round.
+- `description` - a description of the round being run.
+- `txDuration` - the specification of the test duration, in seconds
+- `rateControl` - a rate control type, with options. 
+- `workload` - the workload module to use, with arguments to pass to the module. All arguments passed are available as `roundArguments` within the workload module.
+
+We will specify a benchmark round labeled `queryAsset`, with the description `Query asset benchmark`, to run for a 30s duration, using a `fixed-load` rate controller aiming to maintain a constant transaction pressure of 2. Additionally we will be providing a workload through specification of our `queryAssetBenchmarks.js` workload file, which we will pass the arguments `{assets: 10, contractId: asset-transfer-basic}`.
```

Comment:
queryAsset and references to query should be changed to readAsset. The workload module is called `readAsset.js` not `queryAssetBenchmarks.js`